### PR TITLE
Fix page metrics by using the same start page timing as requests

### PR DESCRIPTION
**Pull request by renajohn**
loadEventStart, loadEventEnd and all other page metrics are based on the navigation start that kick off just after the unload event of the previous page. Requests timing are based on the first outgoing HTTP request. This creates an offset between page timings and requests timing making impossible to compare those metrics.

This PR fixes the time reference used by the page metrics to be the same as the request metrics. To do so, browser agents are passing absolute time in milliseconds since UNIX EPOCH instead of relative time. Then the relative time is computed by the WPT Hook just before persisting the page information on disk.

### CODE REVIEW

File: agent/wpthook/results.cc [hunk 1]
```diff
@@ -483,15 +483,36 @@ void Results::SavePageData(OptimizationChecks& checks){
     CStringA result;
     CStringA buff;
 
+    // The browser agent sends metrics in milliseconds since UNIX EPOC. We need to convert the start time to
```
Comment (ocr) [inline agent/wpthook/results.cc]: EPOC -> EPOCH

File: agent/wpthook/results.cc [hunk 2]
```diff
@@ -483,15 +483,36 @@ void Results::SavePageData(OptimizationChecks& checks){
     CStringA result;
     CStringA buff;
 
+    // The browser agent sends metrics in milliseconds since UNIX EPOC. We need to convert the start time to
+    // a UNIX timestamp in MS and account for the offset applied to the requests (_start)
+
+    // we need a UNIX EPOC of start time in ms. So,
+    // 1. lets convert the FILETIME to a 64 LONGLONG
+    LONGLONG ll_start = (LONGLONG)_test_state._start_time.dwLowDateTime + ((LONGLONG)(_test_state._start_time.dwHighDateTime) << 32LL);
```
Comment (ocr) [inline agent/wpthook/results.cc:491]: I may be wrong, but I believe you should be able to cast _start_time directly to ll_start:

```
LONGLONG ll_start = (LONGLONG)_test_state.start_time;
```

Comment (renajohn) [inline agent/wpthook/results.cc:491]: The compiler refuses.

I think my implementation is the correct way of doing a cast to LONG LONG.
https://support.microsoft.com/en-us/kb/188768

**Comment by renajohn** [comment]
All comments addressed.

**Comment by ocr** [comment]
:+1:

**Pull request merged**